### PR TITLE
Mikael

### PR DESCRIPTION
**Pull request by wmmihaa**

### CODE REVIEW

File: nav/_integration/working-with-incident-policies.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  Working with incident policies
+description: Incident policies is part of Device Management and allow you to take actions on exceptions and alerts, such as when Nodes comes of line or custom alerts.
```
Comment (patrikSohlman) [inline nav/_integration/working-with-incident-policies.md:4]: Nodes should be cursive

File: nav/_integration/working-with-incident-policies.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Apart from custom policies, there are two policies enabled by default which are **Unhanded Exceptions** and **Offline Nodes** and are described in detail below.
+> **Incident Policies** is only available on managed *Organizations* and can be enabled on the [Organization page](https://microservicebus.com/organizations/detail).
+
+## What is an incident?
+An *incident* can be created from anywhere but most commonly from a *Node* and is commonly some abnormality you’d like to take action on. All *incidents* have an identifier (error code), a description and an action. Actions are described in detail (below)[#Actions].  
+
+## Create a custom alert
+A custom alert is implemented using an **Incident Policy** which defines three elements: The *identifier* (error code), description and an action. Navigate to the [Organization page](https://microservicebus.com/organizations/detail), and scroll down to the *ServiceNow* section. If your organization is managed, the *MANAGE INCIDENT POLICIES* button is enabled. Click the button to open the *Incident policies* dialog.
```
Comment (patrikSohlman) [inline nav/_integration/working-with-incident-policies.md:17]: organization/details in link

File: nav/news.md [hunk 3]
```diff
@@ -11,8 +11,16 @@ summary{
 }
 </style>
 
+<details open><summary markdown="span">August 3rd, 2019</summary> 
 
-<details open><summary markdown="span">July 19th, 2019</summary> 
+### [microservicebus.com](https://microservicebus.com)
+
+* Manage Incident policies
+    * *Incident policies is part of Device Management and allow you to take actions on exceptions and alerts, such as when Nodes comes of line or custom alerts. For more info visit [docs.microservicebus.com](https://docsmicroservicebus.com/working-with-incident-policies)*
```
Comment (patrikSohlman) [inline nav/news.md:19]: Nodes cursive

File: nav/_integration/working-with-incident-policies.md (hunk 1 above)
Comment (wmmihaa) [reply nav/_integration/working-with-incident-policies.md:4]: in description

**Review by patrikSohlman — APPROVED** [review]

**Pull request merged**